### ISSUE TITLE
[bug] Random module already defined error(s)

### ISSUE THREAD

**Issue by janhancic** [BladeRunnerJS/brjs#1522]
I was working on a new feature in our app and refreshing the app as I was doing it. Then one time after I refreshed I got this errors in my JS console:

![image](https://cloud.githubusercontent.com/assets/356488/9848314/fa0dee7c-5ad7-11e5-9537-87d37d93cb10.png)

I then refreshed the app and they disappeared and everything was A-OK.

To be clear, I wasn't working on anything mentioned in any of those errors and the app was working fine after a refresh, without me changing anything.

**Comment by andyberry88** [comment]
Are you able to reproduce this on demand or is it something that happens very occasionally?

**Comment by janhancic** [comment]
Saw it twice now, it happens randomly, so no, I can't reproduce it on demand.

**Comment by andyberry88** [comment]
What IDE are you using? @hobojed saw something similar to this, albeit it server side, when using WebStorm/Intelij since it saves a temporary copy of the file in a parallel folder structure.

**Comment by janhancic** [comment]
I'm using SublimeText 3.

**Comment by janhancic** [comment]
I'm seeing this more and more frequently now.

**Comment by andyberry88** [comment]
Is there anything that seems to make it more common or reproducible? AFAIK no one else has seen this so it'll be near impossible to debug it.

**Comment by janhancic** [comment]
Nope, can't put my finger on anything. I'm just doing my work normally same as I always did.

**Comment by janhancic** [comment]
Got it again just now and did some digging.

My bundling mode is set to none.
Then I looked at one of the modules that I was getting an error for and then noticed (in the network tab) that it was downloaded 3 times!

![image](https://cloud.githubusercontent.com/assets/356488/10102469/8e40a05a-6396-11e5-9f45-bc5e6f1aa58a.png)

I then inspected the DOM and saw that the script tag for this file was there twice (so not sure why it was downloaded 3 times).

**Comment by andyberry88** [comment]
Thanks @janhancic that gives us something to go on. It might be the tag handler plugin that's printing the script tags isn't using a `set` so assets end up being output twice.

**Comment by dchambers** [comment]
@janhancic has noticed that all of the classes in his app are being loaded twice (three times when he saw the error messages). We've been able to confirm that this seems specific to:

  1. SalesTrader (didn't occur on DevTrader or the dashboard)
  2. Chrome (didn't occur on Firefox)
  3. UNIX (didn't occur on Windows, but did occur on Mac & Linux)

Possibly fixing this easy to replicate issue will also fix the occasional error messages he is seeing too.

**Comment by andyberry88** [comment]
> Chrome (didn't occur on Firefox)

Can you `view-source` for the `index.html` in Chrome to see if requests are added twice in the source. That should (dis)prove whether Chrome is doing something weird.
